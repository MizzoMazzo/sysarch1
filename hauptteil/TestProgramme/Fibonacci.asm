addiu $1, $0, 0 # f(n-1)
addiu $2, $0, 1 # f(n)
loop:
addu $3, $2, $1
addu $1, $2, $0
addu $2, $3, $0
j loop
