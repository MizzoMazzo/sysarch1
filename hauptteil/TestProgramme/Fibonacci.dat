0x24010000
0x24020001
0x00411821
0x00400821
0x00601021
0x08100002
