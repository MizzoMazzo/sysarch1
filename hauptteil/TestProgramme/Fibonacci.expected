@1 d
@2 15
@3 15
